include/ringmesh/geomodel/core/geomodel.h

```diff
@@ -74,9 +74,9 @@ namespace RINGMesh
     FORWARD_DECLARATION_DIMENSION_CLASS( GeoModelBuilderBase );
     FORWARD_DECLARATION_DIMENSION_CLASS( GeoModelBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( GeoModelBuilderGM );
-
     ALIAS_2D_AND_3D( GeoModelMeshEntity );
     ALIAS_2D_AND_3D( Region );
+    class StratigraphicColumn;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant forward declaration; the declaration was already made. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3231)